src/smtp.rs

```diff
@@ -442,7 +446,14 @@ pub(crate) async fn send_msg_to_smtp(
                         None,
                         None,
                     )
-                    .await?;
+                    .await
+                    .context("add_info_msg_with_cmd")
+                    .log_err(context)
+                    .is_err()
+                    {
+                        // The chat might be deleted in parallel.
+                        return Ok(());
```

Comment:
We shouldn't remove it because we don't check the exact error. If the chat was really deleted, on the next iteration we won't get here and delete the `smtp` row

Reply:
Yes, then i will revert this part, but also `add_info_msg_with_cmd()` shouldn't fail just because the chat doesn't exist, it should return `Option<MsgId>` instead

Reply:
It doesn't fail, it just inserts a message to the deleted chat, but that's ok for this code